source/renderer/app/components/sidebar/wallets/SidebarWalletMenuItem.js

```diff
@@ -40,6 +43,15 @@ export default class SidebarWalletMenuItem extends Component<Props> {
           <span className={styles.title}>{title}</span>
           <span className={styles.info}>{info}</span>
           {isRestoreActive ? <ProgressBar progress={restoreProgress} /> : null}
+          {isLegacy && (
+            <LegacyBadge
+              style={{
```

Comment:
@yakovkaravelov why do we have a style prop here? Can't we sort out styling in the component itself?